pipelines/rbsa/rbsa.py

```diff
@@ -1,6 +1,7 @@
 import logging
 from pipelines.rbsa.tasks import normalize
 from pipelines.rbsa.tasks import group_sites
+from pipelines.rbsa.tasks import index_heatcool
```

Comment:
I know this is still in DRAFT, but you should be able to just comma separate these imports:
```
from pipelines.rbsa.tasks import group_sites, index_headcool
```

Reply:
fixed